pkg/webhook/scheduler/mutating/utils.go

```diff
@@ -25,8 +25,10 @@ func buildRuntimeInfoInternal(client client.Client,
 		namespace = corev1.NamespaceDefault
 	}
 	pvcName := pvc.GetName()
-	if wrapperName, exists := pvc.Labels[common.LabelAnnotationWrappedBy]; exists {
-		pvcName = wrapperName
+	if datasetName, exists := pvc.Labels[common.LabelAnnotationManagedBy]; exists {
```

Comment:
How about creating a function to check this?

Reply:
Done.